Source/Classes/PINURLSessionManager.m

```diff
@@ -149,6 +152,8 @@ - (void)URLSession:(NSURLSession *)session task:(NSURLSessionTask *)task didComp
                                     userInfo:@{NSLocalizedDescriptionKey : @"HTTP Error Response."}];
         }
     }
+    
+    if (delegateQueue == nil) return;
```

Comment:
Do you mind adding this assert above this line and all the other nil checks?:
`NSAssert(delegateQueue != nil, @"There seems to be an issue in iOS 9 where this can be nil. If you can reliably reproduce hitting this, *please* open an issue: https://github.com/pinterest/PINRemoteImage/issues");`